### ISSUE TITLE
docbook example seems to be showing me errors

### ISSUE THREAD

**Issue by uncomfyhalomacro** [sile-typesetter/sile#1657]
New user of sile and reading the Sile book. I am using version 0.14.6 of Sile in openSUSE Tumbleweed.

Page 12 says to run `sile --class docbook docbook.xml`. I was expecting it will run flawlessly but it showed me this:

```sh
❯ sile --class docbook docbook.xml
SILE v0.14.5 (Lua 5.4)
<docbook.xml> as xml

! This isn't a SILE document! at docbook.xml: in <snippet>:
                [[<?xml version="1.0" encoding="utf-8"?>  <!-- -*- nxml -*- -->␍␤<!DOCTYPE article [␍␤<!ENTITY version]]

! Unexpected Lua error

error summary:
        Processing at: docbook.xml: in <snippet>:
                [[<?xml version="1.0" encoding="utf-8"?>  <!-- -*- nxml -*- -->␍␤<!DOCTYPE article [␍␤<!ENTITY version]]
        Using code at: /usr/share/sile/outputters/libtexpdf.lua:30: attempt to index a nil value (field 'paperSize')

Run with --traceback for more detailed trace leading up to errors.
```

I am not sure if this is a Sile issue or not.

**Comment by Omikhleia** [comment]
From your logs, you are running 0.14.**5** which a similar issue (#1637), fixed in 0.14.6

**Comment by uncomfyhalomacro** [comment]
Oh i didn't notice. i was looking at publishing instead of factory in opensuse. publishing already ships 0.14.6 while factory is still with 0.14.5. so i have 0.14.5. my bad. I will install 0.14.6. thanks!.
